package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "topcoat-public",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Can we replace this script with:

`"prettify": "npx prettier --write ."`

?

Reply:
Agree, this will be very convenient if works.

Reply:
This will also replace the need for my PR.

Reply:
@gau94rav and @msorber I added those scripts. I tried to break them down the way they are in app-ui/product-cl though: 
https://github.com/PatchUI/ProductCL/blob/main/package.json
https://github.com/snyk/app-ui/blob/main/package.json

I also used `npm` instead of `npx` 'because'. I don't know if using `npx` is better 🤷‍♂️  I guess if we used npx we wouldn't need to install the packages. It's a tradeoff.